node-notes-cli/app.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-console */
+const fs = require('fs');
+const json = require('./data.json');
+let count = 0;
+
+if (process.argv[2] === 'read') {
+  console.log(JSON.stringify(json.notes, null, 2));
+} else if (process.argv[2] === 'create') {
+  const arr = [];
+  for (const key in json.notes) {
+    count++;
+    arr.push(Number(key));
+  }
+  count = Math.max(...arr); // this allows 'create' to function properly after the user used 'delete' on an entry.
+  json.notes[count + 1] = process.argv[3];
+  fs.writeFile('data.json', JSON.stringify(json, null, 2), 'utf8', () => console.log(`data.json updated, you now have ${count + 1} notes.`));
```

Comment:
Be sure to at least re-throw the `err` if the writeFile fails. It's not good to ignore the `err`.